deps/llvm.mk

```diff
@@ -175,6 +175,19 @@ LLVM_CPPFLAGS += -flto
 LLVM_LDFLAGS += -flto
 endif # LLVM_LTO
 
+ifeq ($(BUILD_CUSTOM_LIBCXX),1)
+LLVM_LDFLAGS += -Wl,-R$(build_libdir)
+LLVM_CXXFLAGS += -I$(build_includedir)
+LLVM_CPPFLAGS += -I$(build_includedir)
+LLVM_CFLAGS += -I$(build_includedir)
```

Comment:
We probably don't need it in all of them, I was just playing it safe because FreeBSD sucks at figuring out where to search for things, plus I figured it wouldn't hurt. I'll try paring it down to just `CPPFLAGS`.

As an aside, what's the difference between `CPPFLAGS` and `CXXFLAGS`?